CMakeLists.txt

```diff
@@ -3,39 +3,31 @@
 # Author:       Denis Gagnon                  <gagnon88@gmail.com>            #
 # Date:         2015-06-01                                                    #
 # Description:  CMake compilation instructions.                               #
+# Author:       Konstantin Ladutenko          <kostyfisik@gmail.com>          #
+# Date:         2021-03-24                                                    #
+# Description:  CMake3 update, using cmake config from                        #
+#               https://github.com/szaghi/PENF as a reference.                #
 # ----------------------------------------------------------------------------#
 
-# ----------------------------------------------------------------- #
-# --                 Name and version of library                 -- #
-# ----------------------------------------------------------------- #
-project(complex_bessel)
-set (complex_bessel_VERSION_MAJOR 0)
-set (complex_bessel_VERSION_MINOR 7)
-set (complex_bessel_VERSION_RELEASE 0)
-
-# ----------------------------------------------------------------- #
-# --               Configuration and Dependencies                -- #
-# ----------------------------------------------------------------- #
-# -- CMake version and installation directory.
-cmake_minimum_required(VERSION 2.8)
-if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-    set (CMAKE_INSTALL_PREFIX /usr)
-endif()
-
-option(BUILD_STATIC_LIBS "Builds static libraries as well as shared libraries" OFF)
-option(BUILD_TESTING "Build the testing tree" OFF)
-
-# ----------------------------------------------------------------- #
-# --                    Compiler Configuration                   -- #
-# ----------------------------------------------------------------- #
-
-# -- Compiler config (flags and such)
-enable_language (Fortran)
-
-# -- Default build type
-if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE Release)
-endif(NOT CMAKE_BUILD_TYPE)
+# The target of the complex_bessel library is exported
+# as complex_bessel::complex_bessel to a package configuration file for this
+# library
+#
+# usage:
+#     find_package(complex_bessel) or add_subdirectory(complex_bessel)
+# and then for your <target>:
+#     target_link_library(<target> complex_bessel::complex_bessel)
+#
+# the config file is generatet in the build and install directories
```

Comment:
typo: "generated"

Reply:
I know about this typo, if you don't mind I would keep it. The reason is that the initial example CMakeList.txt has this typo, so it is kind of a fingerprint to show this origin. I know a dozen of other CMake projects wich keep this typo.

Reply:
You've already put the link in the header. I'd rather have the typo fixed.

Reply:
done